cmake_modules/rocjpeg-config.cmake.in

```diff
@@ -11,7 +11,9 @@ set_and_check(rocjpeg_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
 set_and_check(ROCJPEG_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
 set_and_check(rocjpeg_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
 set_and_check(ROCJPEG_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")
-set_and_check(ROCM_PATH "${PACKAGE_PREFIX_DIR}")
+set_and_check(rocjpeg_LIBRARY "@PACKAGE_LIB_INSTALL_DIR@/librocjpeg.so")
+set_and_check(ROCJPEG_LIBRARY "@PACKAGE_LIB_INSTALL_DIR@/librocjpeg.so")
+
```

Comment:
Please add the following two lines to support find_package(rocJPEG). Some components (e.g., profiler) use rocJPEG to find the rocJPEG package; therefore, we need to support this variant as well.

```
set_and_check(rocJPEG_INCLUDE_DIR "@PACKAGE_INCLUDE_INSTALL_DIR@")
set_and_check(rocJPEG_LIB_DIR "@PACKAGE_LIB_INSTALL_DIR@")

```